dev-libs/wlroots/wlroots-0.1.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/swaywm/wlroots.git"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/swaywm/wlroots/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+inherit fcaps meson
+
+DESCRIPTION="Pluggable, composable, unopinionated modules for building a Wayland compositor"
+HOMEPAGE="https://github.com/swaywm/wlroots"
+LICENSE="MIT"
+SLOT="0"
+IUSE="elogind examples filecaps png rootston systemd x11-backend xcb-icccm xcb-xkb xwayland"
+REQUIRED_USE="?? ( systemd elogind )"
+
+# An example requires >=ffmpeg-4.0 (actually masked) or equivalent libav
+# (actually upstream is behind ffmpeg-4.0). So the ffmpeg use flag is actually
+# not present and the related example (only one) will not be build. This is a
+# reminder to add the 'ffmpeg' use flag when ffmpeg-4.0 will be unmasked.
+# examples? ( ffmpeg? ( >=media-video/ffmpeg-4.0 ) )
+RDEPEND=">=dev-libs/libinput-1.7.0:0=
+	>=dev-libs/wayland-1.16.0
+	>=dev-libs/wayland-protocols-1.15
+	media-libs/mesa[egl,gles2,gbm]
+	virtual/libudev
+	x11-libs/libdrm
+	x11-libs/libxkbcommon
+	x11-libs/pixman
+	filecaps? ( sys-libs/libcap )
+	elogind? ( >=sys-auth/elogind-237 )
+	examples? (
+		png? ( media-libs/libpng:* )
+	)
+	systemd? ( >=sys-apps/systemd-237 )
+	x11-backend? (
+		x11-libs/libxcb:0=
+		xcb-xkb? ( x11-libs/libxcb[xkb] )
+	)
+	xcb-icccm? ( x11-libs/xcb-util-wm )
+	xcb-xkb? ( x11-libs/libxcb[xkb] )
+	xwayland? (
+		x11-base/xorg-server[wayland]
+		x11-libs/libxcb:0=
+	)"
+DEPEND="${RDEPEND}"
+BDEPEND=">=dev-util/meson-0.48
```

Comment:
as I said you do not need to state meson dependency explicitly, it is inherited from the meson eclass.

Reply:
Oh ops.. I'll fix in a min

Reply:
But there's a problem: `meson.eclass` actually adds dependency on `>=dev-util/meson0.45.1` while wlroots require v0.48 or higher: https://github.com/swaywm/wlroots/blob/c55d1542fe30ea7872a60a732fa88028cd4d4b06/meson.build#L6

Reply:
I've added back the explicit dependency again, since this is the approach also followed by [sys-apps/systemd-9999](https://github.com/gentoo/gentoo/blob/960210d67a8a1bd7c32493a4a3831d01efd98bef/sys-apps/systemd/systemd-9999.ebuild#L91).